pulsar-client-tools-test/src/test/java/org/apache/pulsar/admin/cli/CmdFunctionsTest.java

```diff
@@ -424,6 +425,27 @@ public void testCreateFunctionWithInvalidPackageUrl() throws Exception {
         verify(functions, times(0)).createFunctionWithUrl(any(FunctionConfig.class), anyString());
     }
 
+    @Test
+    public void testCreateFunctionWithBuiltinNar() throws Exception {
+        cmd.run(new String[] {
+                "create",
+                "--name", FN_NAME,
+                "--inputs", INPUT_TOPIC_NAME,
+                "--output", OUTPUT_TOPIC_NAME,
+                "--jar", BUILTIN_NAR,
```

Comment:
this is not how builtin sources and sinks work.

with Sinks and Sources you use "--sink-type" to refer to a builtin Sink. This way we are not depending on the name of the jar file

Reply:
with Sinks there is no need to pass the classname

Reply:
> this is not how builtin sources and sinks work.
with Sinks and Sources you use "--sink-type" to refer to a builtin Sink. This way we are not depending on the name of the jar file

For Sinks/Source you can also give `--archive builtin://mysink`.
It would be useful to have a `--function-type` param but that I think it should be another PR

Reply:
> with Sinks there is no need to pass the classname

Yes this should not be required. Do I fix it in this PR or another one ?